site/docs/configuration.md

```diff
@@ -84,7 +84,7 @@ Iceberg catalogs support using catalog properties to configure catalog behaviors
 | warehouse                         | null               | the root path of the data warehouse                    |
 | uri                               | null               | a URI string, such as Hive metastore URI               |
 | clients                           | 2                  | client pool size                                       |
-
+| cache-enabled                     | false              | cache catalog, only works for Spark                    | 
```

Comment:
There is also a `cache-enabled` option in [FlinkCatalogFactory](https://github.com/apache/iceberg/blob/master/flink/src/main/java/org/apache/iceberg/flink/FlinkCatalogFactory.java#L129). I have created a PR about this: #2648